packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js

```diff
@@ -166,7 +166,7 @@ export function PatternCategoryPreviews( {
 						variant="muted"
 						className="block-editor-inserter__patterns-category-no-results"
 					>
-						{ __( 'No results found' ) }
+						{ __( 'No results found.' ) }
```

Comment:
This change seems unnatural. Is it intentional? I think block names should be nouns.

<img width="287" height="236" alt="Image" src="https://github.com/user-attachments/assets/081b0134-93e6-4372-8d20-02dab6383324" />

Reply:
I'm not sure which would be best, sorry.

There's an old journalism saying "if in doubt leave it out" that I think applies here so I've reverted the change in 3a9853eb3358d93762c202d5485845e233b3dd1a 

@annezazu may have a better idea of the inline docs/copy standards here.